### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sedkis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,6 @@ See more at:
 
 Full deployment examples are available in the deployments directory.
 
+## Compatibility
+These modules were written for Terraform version 0.11.  Version 0.12 has backwards incompatible syntax and may not work without modifications.
```
Comment (excieve) [inline README.md]:
```suggestion
These modules were written for Terraform version 0.11. Version 0.12 has backwards incompatible syntax and may not work without modifications. Migration to Terraform 0.12+ is planned, see #2.
```

**Review by excieve — APPROVED** [review]
Thanks! Just a slight suggestion to reference the 0.12 ticket.

**Comment by sedkis** [comment]
@excieve fixed changes

**Comment by sedkis** [comment]
@excieve go ahead and merge please

**Pull request merged**